internal/statesync/syncer_test.go

```diff
@@ -724,6 +721,8 @@ func TestSyncer_applyChunks_RejectSenders(t *testing.T) {
 
 func TestSyncer_verifyApp(t *testing.T) {
 	boom := errors.New("boom")
+	appVersion := uint64(9)
```

Comment:
```suggestion
	const appVersion = 9
```

I don't think this passes through an empty interface so I think it's fine not to state the type.